content/03.methods.md

```diff
@@ -19,5 +19,18 @@
 ### Gene Expression Abundance Estimation
 
 ### RNA Fusion Calling and Prioritization
+**Gene Fusion Detection**
+> Arriba v1.1.0 and STAR-Fusion 1.5.0 fusion detection tools were set up using CWL on CAVATICA.
```

Comment:
Is this a quote? I'm not sure why the carat symbols are here. Also, can you add references for the tools?

Reply:
I think we should also link to the github code for these analyses - @cgreene should we link to each analysis separately or link to the general github for KidsFirst? Should we do it within the text or have a table - this may get long.

Reply:
You can make things a link by enclosing them with brackets and using parens to provide the URL. Linking to the specific source code seems ideal since it's possible and we'd like this to be an exemplar of how to do open, reproducible science around a large collection of samples.

Reply:
these have been referenced and github code linked